events/models.py

```diff
@@ -1,6 +1,29 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+
+
 from django.db import models
 
 # Create your models here.
+
+
+class Event(models.Model):
+    Name_of_event = models.CharField(max_length=100)
+    Starting_date = models.DateTimeField()
+    Ending_date = models.DateTimeField()
```

Comment:
All the column names should start lowercase. Rename name_of_event to name and increase its max length. description doesn't need a max_length. mentor_name and created_by should be Foreign key of User. No need of mentor_contact and mentor_email. Rename beginner_level to just level and create Choices for beginner, intermediate and expert. reference : https://docs.djangoproject.com/en/1.11/ref/models/fields/